### PR TITLE
react-native docs

### PR DESCRIPTION
**Pull request by aiba**
Documenting how to get started with helix + react-native.

### CODE REVIEW

File: docs/react-native.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+```clojure
+:devtools {:preloads [helix.experimental.refresh]
+           :reload-strategy :full}
+```
+
+It's an open question whether `{:reload-strategy :full}` is strictly necessary to
+emulate the React Fast Refresh behavior, but it doesn't seem to hurt.
+
+Then, we just need to hook into Fast Refresh from the main part of our app, as well
+as add `{:helix/features {:fast-refresh true}}` to every component:
```
Comment (Lokeh) [inline docs/react-native.md]: We should add a link to the ["create a custom macro"](https://github.com/Lokeh/helix/blob/master/docs/pro-tips.md#create-a-custom-macro) section in pro-tips.

**Comment by aiba** [comment]
Good call, link to custom macro pro-tip added.

File: docs/react-native.md (hunk 1 above)
Comment (aiba) [reply docs/react-native.md]: Good call, link to custom macro pro-tip added.

**Comment by Lokeh** [comment]
Thanks! 🚀

**Pull request merged**